src/OrchardCore.Modules/OrchardCore.BackgroundTasks/Views/BackgroundTask/Edit.cshtml

```diff
@@ -38,7 +38,7 @@
                 <div class="@Orchard.GetEndClasses()">
                     <div class="input-group mb-3">
                         <input asp-for="Schedule" class="form-control" />
-                        <button id="use-default-schedule" class="btn btn-outline-success">@T["Reset Schedule"]</button>
+                        <span id="use-default-schedule" class="btn btn-outline-danger">@T["Reset Schedule"]</span>
```

Comment:
The element being a button semantically causes it to capture the enter keypress and resets the value instead of submitting the form.

Reply:
How about an `a`? Having a clickable `span` is more of a workaround than having a usage-appropriate element. Do you know why the rest happens in the first place, though? I don't see from the code.

Reply:
Did you try setting the  type attribute of the button to `button` so it is not defaulting to submit?

```
<button type="button" id="use-default-schedule" class="btn btn-outline-success">@T["Reset Schedule"]</button>
```

Reply:
That didn't occur to me, but yes, `<button type="button" ...>` works! https://www.w3schools.com/tags/tag_button.asp doesn't say that it would be the default, but then it probably it is, because I didn't find any event handlers (my first guess was something from Bootstrap) that would cause this.

Reply:
It's most browser default behavior not bootstrap https://www.w3.org/TR/html401/interact/forms.html#h-17.5